### ISSUE TITLE
Gamepad trigger buttons (R2/L2) implementation

### ISSUE THREAD

**Issue by gandalfthewhite19890404** [shadps4-emu/shadPS4#461]
R2 and L2 implemented with bugs - after pressing and release it does not freeing.
You can check it with RE 4, RDR easily - press L2 to aim and then R2 to shoot, player will shoot all ammo because of stucked (non-freed) R2.

**Comment by georgemoralis** [comment]
It is only gamepad issue or it happens with keyboard as well?

**Comment by bigol83** [comment]
I checked with keyboard on Red Dead Redemption and L2 and R2 buttons work fine. I checked with a Dualshock 4 and they also work fine, so i can't reproduce the issue at least with Red Dead Redemption,

I checked on latest main. 

Checked Dark Souls remastered with Dualsense and i can confirm there are issues with it. L2 and R2 input only work one time, after that pressing L2 and R2 input is not registered, it's like i have not pressed them. The Dark Souls remastered issues with L2 and R2 happen with both Dualshock 4 and Dualsense for me.
